src/main/java/com/rakovets/course/java/core/practice/reflection/Model.java

```diff
@@ -0,0 +1,17 @@
+package com.rakovets.course.java.core.practice.reflection;
+
+public class Model {
+    private String name;
+
+    public Model(String name){
+        this.name = name;
+    }
+
+    public String getName() {
+        return name;
+    }
+
+    public void printName(){
```

Comment:
"Violation: There isn't separating white spaces for curly brake
Solution: Add white spaces"